### ISSUE TITLE
mvn package per your instructions does not work

### ISSUE THREAD

**Issue by gregrluck** [microprofile/microprofile.io#62]
There is no pom file.

**Comment by karianna** [comment]
Yeah the docs need to be updated and the repos renamed.  The gitpress repo is the real site.  @tveronezi Are you able to make the changes after J1 so this all makes a bit more sense?
